### PR TITLE
Let Masque Skin the Glow

### PR DESCRIPTION
**Pull request by Zinoh**
Allows Masque to handle the Glow which inludes a custom glow for Circular Icons.

### CODE REVIEW

File: WeakAuras/RegionTypes/Icon.lua [hunk 1]
```diff
@@ -538,13 +538,20 @@ local function modify(parent, region, data)
...
   function region:SetGlow(showGlow)
-    if (showGlow) then
-      if (not region.__WAGlowFrame) then
-        region.__WAGlowFrame = CreateFrame("Frame", nil, region);
-        region.__WAGlowFrame:SetAllPoints();
-        region.__WAGlowFrame:SetSize(region.width, region.height);
-      end
-      WeakAuras.ShowOverlayGlow(region.__WAGlowFrame);
+    if MSQ then
+        if (showGlow) then
```
Comment (emptyrivers) [inline WeakAuras/RegionTypes/Icon.lua]: Your indentation needs work. 2 spaces is the format we use

File: WeakAuras/RegionTypes/Icon.lua [hunk 2]
```diff
@@ -538,13 +538,20 @@ local function modify(parent, region, data)
...
+            WeakAuras.ShowOverlayGlow(region.button);
+        else
+            WeakAuras.HideOverlayGlow(region.button);
+        end
+
```
Comment (emptyrivers) [inline WeakAuras/RegionTypes/Icon.lua]: extra newline

**Comment by Stanzilla** [comment]
Hey there, thanks for the contribution! I just fixed your indentation and merged it!

**Comment by emptyrivers** [comment]
that works too

**Comment by Zinoh** [comment]
Yeah sorry about that. I'm not a programmer but followed the wiki as best I could. Set up vs-studio with the extensions mentioned so thought it was a good sign I didn't trigger any warnings :). Thanks for fixing it!

**Pull request merged**